flag-app/flag-app-project/src/components/CardWrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import useCountries from "../hooks/useCountries";
+import Card from "./Card";
+import ErrorMessage from "./ErrorMessage";
+
+interface Props {
+  searchInput: string;
+}
+
+const CardWrapper = ({ searchInput }: Props) => {
+  const { countries, error, isLoading } = useCountries(searchInput);
+
+  if (isLoading) console.log(isLoading);
```

Comment:
`console.log` statements should never exist in "production" code. Also, you should return JSX if `isLoading` is true - it's usually some kind of loading spinner.

Reply:
Yes, I am planning to create some skeletons or loader spiners for it. I leaved the console.log there so I do not have this error message: 'isLoading' is assigned a value but never used.

Is there some good way how to handle this (other than console.log)?